app/reducers/optional.js

```diff
@@ -1,7 +0,0 @@
-//this is a simple pass thru that offers an extension point
```

Comment:
Currently because you're missing a default `index.js` the first thing a user experiences is an error:

![2016-09-15_22-06-27](https://cloud.githubusercontent.com/assets/980757/18567628/f8c7b036-7b90-11e6-9ee3-dbe1fb1da07f.png)